### PR TITLE
some basic docker usage

### PR DESCRIPTION
**Pull request by reusr1**
some hints about how to use docker

### CODE REVIEW

**Review by GastonGonzalez — CHANGES_REQUESTED** [review]
Please add Docker remove commands to allow testing new branches.

**Review by reggie7 — APPROVED** [review]

**Comment by reusr1** [comment]
> Please add Docker remove commands to allow testing new branches.

@GastonGonzalez I added the docker remove with -rm to the run command. I added a bit of explanation to each command to ease people without a lot of docker knowledge into it a bit more.

File: docs/public/docker/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+```bash
+docker pull peregrinecms/peregrinecms:develop-sling12
+```
+
+## Check out a PR with docker
+
+replace the {number} in the command below with the PR number for the change you would like to inspect
+
+```bash
+docker run -it -p 8080:8080 peregrinecms/peregrinecms:pr-{number}
```
Comment (ehdatheadwire) [inline docs/public/docker/index.md]: Don't we also have to replace 8080 with a made-up number that is greater than 9000? e.g. 9209 for pr-209?

**Comment by ehdatheadwire** [comment]
replace all instances of /peregrinecms with /peregrine-cms -- through testing the new instructions I discovered that this missing dash is what was causing my error messages while testing a pr yesterday :)

**Review by ehdatheadwire — APPROVED** [review]
Recommended adjustments made. Looks good from this view!

**Review by GastonGonzalez — APPROVED** [review]

**Pull request merged**